modules/ultra/pipeline/main.nf

```diff
@@ -0,0 +1,69 @@
+        container "https://depot.galaxyproject.org/singularity/ultra_bioinformatics:0.0.4--pyh5e36f6f_1"
+    } else {
+        container "quay.io/biocontainers/ultra_bioinformatics:0.0.4--pyh5e36f6f_1"
+    }
+
+    input:
+    tuple val(meta), path(reads)
+    path genome
+    path gtf
+
+    output:
+    tuple val(meta), path("outfolder/*.sam")                              , emit: sam
+    tuple val(meta), path("outfolder/database.db")                        , emit: database
+    tuple val(meta), path("outfolder/all_splice_pairs_annotations.pickle"), emit: all_splice_pairs_annotations
+    tuple val(meta), path("outfolder/all_splice_sites_annotations.pickle"), emit: all_splice_sites_annotations
+    tuple val(meta), path("outfolder/chr_to_id.pickle")                   , emit: chr_to_id
+    tuple val(meta), path("outfolder/exon_choordinates_to_id.pickle")     , emit: exon_choordinates_to_id
+    tuple val(meta), path("outfolder/flank_choordinates.pickle")          , emit: flank_choordinates
+    tuple val(meta), path("outfolder/gene_to_small_segments.pickle")      , emit: gene_to_small_segments
+    tuple val(meta), path("outfolder/id_to_chr.pickle")                   , emit: id_to_chr
+    tuple val(meta), path("outfolder/max_intron_chr.pickle")              , emit: max_intron_chr
+    tuple val(meta), path("outfolder/parts_to_segments.pickle")           , emit: parts_to_segments
+    tuple val(meta), path("outfolder/ref_exon_sequences.pickle")          , emit: ref_exon_sequences
+    tuple val(meta), path("outfolder/ref_flank_sequences.pickle")         , emit: ref_flank_sequences
+    tuple val(meta), path("outfolder/ref_part_sequences.pickle")          , emit: ref_part_sequences
+    tuple val(meta), path("outfolder/ref_segment_sequences.pickle")       , emit: ref_segment_sequences
+    tuple val(meta), path("outfolder/refs_id_lengths.pickle")             , emit: refs_id_lengths
+    tuple val(meta), path("outfolder/refs_lengths.pickle")                , emit: refs_lengths
+    tuple val(meta), path("outfolder/segment_id_to_choordinates.pickle")  , emit: segment_id_to_choordinates
+    tuple val(meta), path("outfolder/segment_to_gene.pickle")             , emit: segment_to_gene
+    tuple val(meta), path("outfolder/segment_to_ref.pickle")              , emit: segment_to_ref
+    tuple val(meta), path("outfolder/splices_to_transcripts.pickle")      , emit: splices_to_transcripts
+    tuple val(meta), path("outfolder/transcripts_to_splices.pickle")      , emit: transcripts_to_splices
+    path "versions.yml"                                                   , emit: versions
+
+    script:
+    def prefix = options.suffix ? "${meta.id}${options.suffix}" : "${meta.id}"
+
+    """
+    uLTRA pipeline \\
+        --t $task.cpus \\
+        --prefix $prefix \\
+        $options.args \\
+        \$(pwd)/$genome \\
+        \$(pwd)/$gtf \\
+        \$(pwd)/$reads \\
```

Comment:
Actually yes. The tool do not resolve the file path properly yet, so full path to files is needed.

Reply:
I re-tested this with variations, but I could also only get it to work with \$(pwd) (no ./$gtf, no $gtf)

Reply:
I submitted a [new issue at the ULTRA-repo](https://github.com/ksahlin/ultra/issues/8), with a fix-suggestion. Could be quick, but as I said on the mergathon call, I don't think it's a blocker for merging in - everything else looks good,

Reply:
Thanks for the issue :+1: 
I also think it could be merge in this temporary state. 
In any case, I will update the module as soon as the patch is applied.